libs/tup-components/src/projects/ProjectsListing/ProjectsListing.module.css

```diff
@@ -16,16 +16,23 @@
 
 
 /* Project listing */
+.project-listing {
+  margin-bottom: unset; /* undo core-styles.base.css */
+  padding-left: 10px;
+}
+
 .project-listing-row {
     display: flex;
     flex-direction: row;
     justify-content: space-between;
     gap: 5%;
-    padding-top: 20px;
-    padding-right: 20px;
     padding-bottom: 20px;
-
-    border-bottom: 1px solid #707070;
+}
+.project-listing-row:not(:first-child) {
+    padding-top: 20px;
+}
+.project-listing-row:not(:last-child) {
+    border-bottom: var(--global-border-width--normal) solid var(--global-color-primary--dark);
 }
```

Comment:
**Explanation:** I am only applying the `padding-top` on successive rows, because new section spacing adds space above first row. I am only applying `border-bottom` to the bottom row... because I believe the design was wrong out of neglect to show the bottom row having one.*

<sup>* I'll confirm this... hold on ([private inquiry](https://tacc-team.slack.com/archives/G01K9ET7JM7/p1674077707260529)).</sup>